block-streamer/src/control.rs

```diff
@@ -0,0 +1,50 @@
+use anyhow::Result;
+
+use serde_json::from_slice;
+use std::net::SocketAddr;
+use std::path::PathBuf;
+use std::thread::sleep;
+use std::time::Duration;
+use tokio::net::UdpSocket;
+use tracing::info;
+
+use crate::api::ApplicationAPI;
+use crate::receive::receive;
+use crate::transmit::transmit;
+
+pub async fn control(listen_addr: &String) -> Result<()> {
+    info!("Listening for control messages on {}", listen_addr);
+    let listen_address: SocketAddr = listen_addr.parse()?;
+
+    let mut buf = vec![0; 10240];
```

Comment:
Can you make a comment on the selection of the `10240` value?  You could make it an expressive constant to achieve this explanation as well.

Reply:
I'll address this in #20